lldb/source/Commands/CMakeLists.txt

```diff
@@ -30,6 +30,7 @@ add_lldb_library(lldbCommands NO_PLUGIN_DEPENDENCIES
   CommandObjectSession.cpp
   CommandObjectSettings.cpp
   CommandObjectSource.cpp
+  CommandObjectStart.cpp
```

Comment:
revert this change since we are just adding an option to "process launch" right?